plugin/inc/item.class.php

```diff
@@ -213,4 +213,15 @@ public function additionalGetFromDbSteps($ID = null): void
     {
     }
 
+    public static function generateJsonResponse(string $status = null, string $message = null, string $html = null): string
```

Comment:
(optional)
```suggestion
    public static function getJsonResponse(string $status = null, string $message = null, string $data= null): string
```

1. naming convention
2. we don't always have a html as extra return data

Reply:
2. Well, probably we should add $data later if needed. But now we need html.